library/hawkular_alerts_group_trigger.py

```diff
@@ -134,6 +147,30 @@ def delete_group_trigger(self, group_id):
         except Exception as e:
             self.module.fail_json(msg="Failed to delete group trigger. Error: {error}".format(error=e))
 
+    def conditions_update_required(self, trigger_id, conditions):
+        """ Returns True if an update is required in one or more of the group
+            trigger conditions, False otherwise
+        """
+        tgs = self.client.get_trigger_conditions(trigger_id)
```

Comment:
should have used `tcs`.
Anyway, changed to current_conditions(_sorted) and desired_conditions(_sorted), thanks